### PR TITLE
Add contributing instructions

### PR DESCRIPTION
**Pull request by vladdy**
This document mentions the steps which allowed me to get going on this project. They can also help to others.

Please note that this change depends on Makefile addition from #7  and will need to be adjusted if Makefile request gets rejected.

### CODE REVIEW

**Comment by VojtechVitek** [comment]
This is very useful, thanks for your contribution!

Let's see what others think of #7 first before merging this PR.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+## Prerequests
```
Comment (VojtechVitek) [inline CONTRIBUTING.md]: Prerequisites

**Comment by pkieltyka** [comment]
nice, let's update and merge

**Comment by vladdy** [comment]
Updated instructions to use `go` command and fixed the mistype.

**Comment by pkieltyka** [comment]
I still see `make` in the file

**Comment by vladdy** [comment]
Sorry, missed it in one step.

**Pull request merged**